src/org/omegat/core/Core.java

```diff
@@ -131,6 +132,7 @@ private Core() {
             .synchronizedList(new ArrayList<String>());
 
     private static final List<IMarker> MARKERS = new ArrayList<IMarker>();
+    private static final List<IGlossaryRenderer> GLOS_RENDERERS = new ArrayList<IGlossaryRenderer>();
```

Comment:
No problem for me. The only pattern I tried to follow was consistency with existing code.
I understand that you want to change some patterns in the code of OmegaT, but I think it would be better if you could document it somewhere, instead of saying it individually to developers once they wrote code following old rules. IssueProviders.java is a good reference but its existence is not enough to say that you want to work like that in the future: you must write it somewhere.

Globally I agree with your thoughts about coupling in the message you quote. My reaction in june was not about reducing dependency to Core, but about adding useless dependency between glossaries and all MT engines, while only one really needed it; and after I saw your explanation, once I had all the constraints in mind, I proposed a solution.
And I also emphased in the next message that there are plenty of old code which does not respect the new rules: not only you should document your new wishes, but you could almost change some of old classes, so that you can in your documentation put an example about how we did before and how we do now.

When you say "remove existing stuff", do you mean that you would like to change the pattern for other plugin types as well, and take the risk to break compatibility with external plugins? If so, I would advice that you open a discussion in the developers list. I have some ideas of improvements about the plugins system, but I would better share them in the users list, and preferably in an active discussion, rather than starting it myself and receive no answer because everybody is busy about something else. 

That said, yes, I agree with the principle of this change, I would only have preferred to know it before.

PS. Other questions I will answer later. I wrote this code one year ago, I really need some time to remember what I had in mind at this time.